bootstrap/config/config.go

```diff
@@ -130,15 +141,19 @@ func (cp *Processor) Process(serviceKey string, configStem string, serviceConfig
 
 	switch configProviderInfo.UseProvider() {
 	case true:
-		tokenFile := serviceConfig.GetBootstrap().Service.ConfigAccessTokenFile
-		accessToken, err := token.LoadAccessToken(tokenFile)
-		if err != nil {
-			// access token file doesn't exist means the access token is not needed.
-			if os.IsNotExist(err) {
-				lc.Warnf("Configuration Provider access token at %s doesn't exist. Skipping use of access token", tokenFile)
-			} else {
-				return fmt.Errorf("unable to load Configuration Provider client access token at %s: %s", tokenFile, err.Error())
+		var accessToken string
+
+		// secretProvider will be nil if not configured to be used. In that case, no access token required.
+		if secretProvider != nil {
+			accessToken, err = secretProvider.GetAccessToken(configProviderInfo.serviceConfig.Type, serviceKey)
+			if err != nil {
+				return fmt.Errorf(
+					"failed to get Configuration Provider (%s) access token: %s",
+					configProviderInfo.serviceConfig.Type,
+					err.Error())
 			}
+
+			cp.lc.Infof("Using Config Provider access token of length %d", len(accessToken))
 		}
```

Comment:
request to add cp.lc.Info for else branch like "not configured to use secret provider"

Reply:
Added